astroid/brain/brain_dataclasses.py

```diff
@@ -550,16 +550,6 @@ def _get_field_default(field_call: nodes.Call) -> _FieldDefaultReturn:
     return None
 
 
-def _is_class_var(node: nodes.NodeNG) -> bool:
```

Comment:
I came across this method while looking for a way to detect `ClassVar` annotations, moved it to `utils` to make it reusable, but let me know if there is a more suitable place for it.